aspnet/security/authorization/policies.rst

```diff
@@ -63,7 +64,7 @@ A requirement doesn't need to have data or properties.
 Authorization Handlers
 ----------------------
 
-An authorization handler is responsible for the evaluation of any properties of a requirement. The  authorization handler must evaluate them against a provided ``AuthorizationContext`` to decide if authorization is allowed. A requirement can have :ref:`multiple handlers <security-authorization-policies-based-multiple-handlers>`. Handlers must inherit ``AuthorizationHandler<T>`` where T is the requirement it handles. 
+An authorization handler is responsible for the evaluation of any properties of a requirement. The  authorization handler must evaluate them against a provided :dn:class:`~Microsoft.AspNetCore.Authorization.AuthorizationHandlerContext` to decide if authorization is allowed. A requirement can have :ref:`multiple handlers <security-authorization-policies-based-multiple-handlers>`. Handlers must inherit :dn:class:`~Microsoft.AspNetCore.Authorization.AuthorizationHandler<TRequirement>` where ``TRequirement`` is the requirement it handles. 
```

Comment:
```
:dn:class:`~Microsoft.AspNetCore.Authorization.AuthorizationHandler<TRequirement>`
```

This isn't rendering properly. Is there a specific syntax for generic types I'm missing? See http://vibrantcode.com/AspNetDocsPreview/security/authorization/policies.html#authorization-handlers for example of bad rendering

/cc @danroth27 @RickAndMSFT